### PR TITLE
Ankit | Test - Setting for estimate/proforma and purchase order is not coming (Key change)

### PR DESCRIPTION
**Pull request by ankitdubeyoffice**
https://app.clickup.com/t/86cwtkxjh

### CODE REVIEW

File: apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.html [hunk 1]
```diff
@@ -210,25 +210,25 @@ <h4 class="clearfix bd-rb">{{ localeData?.purchase_order }}</h4>
...
                                                 [(ngModel)]="
-                                                    invoiceSettings.purchaseBillSettings.autoWhatsApp
+                                                    invoiceSettings.purchaseBillSettings.poAutoWhatsApp
                                                 "
                                                 class="show-on-mobile"
-                                                name="autoWhatsApp"
+                                                name="poAutoWhatsApp"
                                                 color="primary"
-                                                aria-label="auto whatsApp"
+                                                aria-label="poAuto whatsApp"
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.html]: purchase order whatsapp

File: apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.html [hunk 2]
```diff
@@ -210,25 +210,25 @@ <h4 class="clearfix bd-rb">{{ localeData?.purchase_order }}</h4>
...
                                                 [(ngModel)]="
-                                                    invoiceSettings.purchaseBillSettings.autoWhatsApp
+                                                    invoiceSettings.purchaseBillSettings.poAutoWhatsApp
                                                 "
                                                 class="hide-on-mobile"
-                                                name="autoWhatsApp"
+                                                name="poAutoWhatsApp"
                                                 color="primary"
-                                                aria-label="auto whatsApp"
+                                                aria-label="poAuto whatsApp"
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.html]: purchase order whatsapp

**Review by dilpreetsinghofficial — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.html (hunk 1 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.html]: Done

File: apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.html (hunk 2 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.html]: Done

**Review by dilpreetsinghofficial — APPROVED** [review]

**Pull request merged**